### PR TITLE
Milestone III: Create Contact Form And Complete Mobile Version - PortFolio Project

### PR DESCRIPTION
**Pull request by ouasamine**
### In this milestone, I completed the following tasks:
- Added contact form to index.
- Styled contact form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -162,5 +162,26 @@ <h2 class="card-header">About me</h2>
     </li>
     </ul>
   </section>
+  <section id="contact">
+    <header>
+      <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h2>
```
Comment (roshan-bajgain) [inline index.html:168]: - [ ]  I would suggest you please increase the font size so that it matches the Figma design.

File: index.html [hunk 2]
```diff
@@ -162,5 +162,26 @@ <h2 class="card-header">About me</h2>
...
+    </header>
+    <form id="contact_form" action="https://formspree.io/f/mayknqdr" target="_self" method="post">
+      <fieldset>
+          <input type="text" name="full_name" placeholder="Full Name" required maxlength="30" />
+          <input type="email" name="email" placeholder="Email address" required />
+          <textarea maxlength="500" name="message" placeholder="Enter text here" required></textarea>
+      </fieldset>
+      <button class="card-button" name="button" type="submit">Get in touch</button>
```
Comment (roshan-bajgain) [inline index.html:176]: - [ ] I would suggest you increase the width of the buttons as `Get in touch` is not showing correctly on the browser try to align the test in the center and provide equal padding from all sides. Please take the image below for reference.
### Your design
your
### Figma design
figma
- [ ] As the project requirement button interactions such as hover, active and pressed should be implemented as in the Figma template.
### Hover template

hover

File: index.html [hunk 3]
```diff
@@ -162,5 +162,26 @@ <h2 class="card-header">About me</h2>
...
+    </form>
+    <ul class="padding-reset social-media">
+      <li><a href=""><img src="./img/Icon-GitHub-b.png" alt="My Github Link"></a></li>
+      <li><a href=""><img src="./img/Icon-Linkedin-b.png" alt="My Linkedin Link"></a></li>
+      <li><a href=""><img src="./img/Icon-Angelist-b.png" alt="My Angelist Link"></a></li>
+      <li><a href=""><img src="./img/Icon-Twitter-b.png" alt="My Twitter Link"></a></li>
+      <li><a href=""><img src="./img/Icon-Medium-b.png" alt="My Medium Link"></a></li>
+    </ul>
```
Comment (roshan-bajgain) [inline index.html:184]: - [ ] I would suggest you increase the size of the icons and please provide a black horizontal line after the icons as shown in image below.
### Your design
hhhhhhhhhhhhhh
### Figma design
ffffffffffffff

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @ouasamine, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (roshan-bajgain) [reply index.html:184]: @ouasamine, It is not compulsory but you can do it as optional if you want. Thanks

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @ouasamine 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights
 - No linter errors. ✔️
 - GitHub flow is followed. ✔️
 - Professional README file. ✔️
 - PR has a good title and summary. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**